src/Compilers/Core/Portable/Operations/OperationExtensions.cs

```diff
@@ -46,6 +46,17 @@ public static IOperation GetRootOperation(this ISymbol symbol, CancellationToken
             }
         }
 
+        public static IEnumerable<ILocalSymbol> GetDeclaredVariables(this IVariableDeclarationStatement declarationStatement)
+        {
+            foreach (IVariableDeclaration group in declarationStatement.Declarations)
```

Comment:
I considered using linq, but compiler code tends to avoid it in general, so I continued the trend.